js/views/modals/Settings/Settings.js

```diff
@@ -30,6 +30,8 @@ export default class extends BaseModal {
       Moderation,
     };
 
+    this.initTab = this.tabViews.hasOwnProperty(opts.initTab) ? opts.initTab : 'General';
```

Comment:
The initTab option allows another view to open the settings modal to a specific tab. It's used here to open the addresses tab.